programs/marginfi/src/state/marginfi_account.rs

```diff
@@ -687,13 +688,14 @@ impl<'info> RiskEngine<'_, 'info> {
         let is_in_isolated_risk_tier = balances_with_liablities.clone().any(|a| {
             // SAFETY: We are shortening 'info -> 'a
             let shorter_bank: &'a AccountInfo<'a> = unsafe { core::mem::transmute(&a.bank) };
-            AccountLoader::<Bank>::try_from(shorter_bank)
+            let risk_tier: RiskTier = AccountLoader::<Bank>::try_from(shorter_bank)
                 .unwrap()
                 .load()
                 .unwrap()
                 .config
                 .risk_tier
-                == RiskTier::Isolated
+                .into();
+            risk_tier == RiskTier::Isolated
```

Comment:
We should be able to get rid of all the changes relating to RiskTier conversion, with the latest separation between u8 enum and explicit padding, right?